backend/compact-connect/tests/app/base.py

```diff
@@ -128,6 +129,70 @@ def _inspect_persistent_stack(
+        api_query_role_logical_id = persistent_stack.get_logical_id(
+            persistent_stack.ssn_table.api_query_role.node.default_child
+        )
+        ssn_table_template = self.get_resource_properties_by_logical_id(
+            persistent_stack.get_logical_id(persistent_stack.ssn_table.node.default_child),
+            persistent_stack_template.find_resources(CfnTable.CFN_RESOURCE_TYPE_NAME),
+        )
+        ssn_key_template = self.get_resource_properties_by_logical_id(
+            ssn_key_logical_id, persistent_stack_template.find_resources(CfnKey.CFN_RESOURCE_TYPE_NAME)
+        )
+        # This naming convention is important for opting into future organization access logging
+        self.assertTrue(ssn_table_template['TableName'].endswith('-DataEventsLog'))
+        # Ensure our SSN Key is locked down by resource policy
+        self.assertEqual(
+            ssn_key_template['KeyPolicy'],
+            {
+                'Statement': [
+                    {
+                        'Action': 'kms:*',
+                        'Effect': 'Allow',
+                        'Principal': {'AWS': f'arn:aws:iam::{persistent_stack.account}:root'},
+                        'Resource': '*',
+                    },
+                    {
+                        'Action': ['kms:Decrypt', 'kms:Encrypt', 'kms:GenerateDataKey*', 'kms:ReEncrypt*'],
+                        'Condition': {
+                            'StringNotEquals': {
+                                'aws:PrincipalArn': [
+                                    {'Fn::GetAtt': [ingest_role_logical_id, 'Arn']},
+                                    {'Fn::GetAtt': [api_query_role_logical_id, 'Arn']},
+                                ],
+                                'aws:PrincipalServiceName': ['dynamodb.amazonaws.com', 'events.amazonaws.com'],
+                            }
+                        },
+                        'Effect': 'Deny',
+                        'Principal': '*',
+                        'Resource': '*',
+                    },
+                    {
+                        'Action': ['kms:Decrypt', 'kms:Encrypt', 'kms:GenerateDataKey*', 'kms:ReEncrypt*'],
+                        'Condition': {'StringEquals': {'aws:SourceAccount': persistent_stack.account}},
+                        'Effect': 'Allow',
+                        'Principal': {'Service': 'events.amazonaws.com'},
+                        'Resource': '*',
+                    },
+                ],
+                'Version': '2012-10-17',
+            },
+        )
+        # Ensure we're using our locked down KMS key for encryption
+        self.assertEqual(
+            ssn_table_template['SSESpecification'],
+            {'KMSMasterKeyId': {'Fn::GetAtt': [ssn_key_logical_id, 'Arn']}, 'SSEEnabled': True, 'SSEType': 'KMS'},
+        )
+        self.compare_snapshot(
+            ssn_table_template['ResourcePolicy']['PolicyDocument'], 'SSN_TABLE_RESOURCE_POLICY', overwrite_snapshot=True
```

Comment:
Should this be `overwrite_snapshot=False`?

Reply:
Yes it should. Will fix.